components/AddressButtons/AddressButtons.tsx

```diff
@@ -12,39 +14,56 @@ const AddressButtons: React.FC<Props> = ({
   address,
   showInspectButton = true,
 }) => {
+  const { network } = useContext(NetworkContext);
+
+  const networkType = network.name === 'MAINNET' ? 'mainnet' : 'testnet';
```

Comment:
you can also just lowercase network.name

Reply:
ill go for lowercase too